app/src/main/java/com/example/lapnguyen/weather_3/api/GPSTracker.java

```diff
@@ -0,0 +1,161 @@
+package com.example.lapnguyen.weather_3.api;
+
+/**
+ * Created by lapnguyen on 16/06/2017.
+ */
+
+import android.Manifest;
+import android.app.AlertDialog;
```

Comment:
ERROR: Unused import: android.app.AlertDialog. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports